src/client/components/AddReservation.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from "react"
+
+export default function AddReservation(props) {
+    const [number_of_guests, setNumber_of_guests] = useState()
+    const [contact_phonenumber, setContact_phonenumber] = useState("")
+    const [contact_name, setContact_name] = useState("")
+    const [contact_email, setContact_email] = useState("")
+    const handleNumber_of_guests = (event) => {
+        let val = event.target.value;
+        const max = props.max_reservations
+        val = val >= max ? 
+            max :
+            val < max && val > 0 ? val : 0;
+        setNumber_of_guests(val)
+    }
+    const handleContact_phonenumber = (event) => {
+        setContact_phonenumber(event.target.value)
+    }
+    const handleContact_name = (event) => {
+        setContact_name(event.target.value)
+    }
+    const handleContact_email = (event) => {
+        setContact_email(event.target.value)
+    }
```

Comment:
Same consistency message.

These functions should be called `handleContactPhoneNumber` and `setContactPhoneNumber` etc